src/etl/htp_metadataset_etl.py

```diff
@@ -123,18 +131,18 @@ def _process_sub_type(self, sub_type):
 
         # This needs to be in this format (template, param1, params2) others will be ignored
         query_list = [
-            [HTPMetaDatasetETL.htp_dataset_query_template, commit_size,
-             "htp_metadataset_" + sub_type.get_data_provider() + ".csv"],
-            [HTPMetaDatasetETL.htp_category_tags_relations_query_template, commit_size,
-             "htp_metadataset_tags_relations_" + sub_type.get_data_provider() + ".csv"],
-            [HTPMetaDatasetETL.htp_dataset_pub_query_template, commit_size,
-             "htp_metadataset_publications_" + sub_type.get_data_provider() + ".csv"],
-            [HTPMetaDatasetETL.htp_pub_relation_template, commit_size,
-             "htp_metadataset_publication_relations_" + sub_type.get_data_provider() + ".csv"],
-            [HTPMetaDatasetETL.htpdataset_xrefs_template, commit_size,
-             "htp_metadataset_xrefs_" + sub_type.get_data_provider() + ".csv"],
-            [HTPMetaDatasetETL.htp_secondaryIds_query_template, commit_size,
-             "htp_metadataset_secondaryIds_" + sub_type.get_data_provider() + ".csv"],
+            [HTPMetaDatasetETL.htp_dataset_query_template,
+             "htp_metadataset_" + sub_type.get_data_provider() + ".csv", commit_size],
+            [HTPMetaDatasetETL.htp_category_tags_relations_query_template,
+             "htp_metadataset_tags_relations_" + sub_type.get_data_provider() + ".csv", commit_size],
+            [HTPMetaDatasetETL.htp_dataset_pub_query_template,
```

Comment:
@christabone can we oneliner these?

Reply:
Sure thing. I'll push the changes. You mean you don't code on a 80x24 terminal?